core/server/data/migrations/versions/4.0/03-populate-status-column-for-members.js

```diff
@@ -0,0 +1,45 @@
+const {chunk} = require('lodash');
+const {createTransactionalMigration} = require('../../utils');
+const logging = require('../../../../../shared/logging');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Updating members.status based on members_stripe_customers_subscriptions.status');
+        const paidMemberIds = (await knex('members')
+            .select('members.id')
+            .innerJoin(
+                'members_stripe_customers',
+                'members.id',
+                'members_stripe_customers.member_id'
+            ).innerJoin(
+                'members_stripe_customers_subscriptions',
+                function () {
+                    this.on(
+                        'members_stripe_customers.customer_id',
+                        'members_stripe_customers_subscriptions.customer_id'
+                    ).onIn(
+                        'members_stripe_customers_subscriptions.status',
+                        ['active', 'trialing', 'past_due', 'unpaid']
+                    );
+                }
+            )).map(({id}) => id);
+
+        // Umm? Well... The current version of SQLite3 bundled with Ghost supports
+        // a maximum of 999 variables, we use one variable for the SET value
+        // and so we're left with 998 for our WHERE IN clause values
+        const chunkSize = 998;
+        const paidMemberIdChunks = chunk(paidMemberIds, chunkSize);
+
+        for (const paidMemberIdsChunk of paidMemberIdChunks) {
+            await knex('members')
+                .update('status', 'paid')
+                .whereIn('id', paidMemberIdsChunk);
+        }
+    },
+    async function down(knex) {
```

Comment:
This down migration makes sense to me. Wanted to confirm there's nothing more that I could be missing, the main reasoning is: we roll back to a `default` value introduced in `02-add-status-column-to-members.js`, otherwise we might end up in partially paid/free members set.

On the other hand, is there even a need to have the down migration as `02-add-status-column-to-members.js` would remove the column completely if rolled back?

Reply:
+1, just wondering the same and if it's better to leave out the down migration than set back the default value once we go forward as setting back incorrect status is probably worse ? 🤷 Again not sure which is better but just thinking out loud.

Reply:
The thinking was that each `down` should leave the db in the state it was before the `up!

Happy to remove the `down` migration though

Reply:
Seeing as you've both approved I'm going to merge as-is, and we can update it if necessary in future